webpack/common.js

```diff
@@ -58,5 +58,13 @@ exports.convertKeysForHtml = function(obj){
 }
 
 exports.getVersion = function(){
-	return execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+	try {
+		const { stdout } = execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+		return stdout;
+	} catch (e) {
+		console.log("Error when getting version from git");
+		console.log("This does not affect build process");
```

Comment:
This should be one line. Something like this:
`Error when getting version from git. This error doesn't affect build process but you won't be able to see current git version of the repository.`